src/Redux/missions/missions.js

```diff
@@ -1,3 +1,5 @@
+import apiCalls from '../../Api/missionsApi'
+const FetchMissions = 'missionStore/missionactions/fetchMissions';
```

Comment:
- [ ] Optional. For the JavaScript best practices. You can define constant variables as uppercase :heavy_check_mark:

Reply:
Done! Thanks !